recipes-core/images/core-image-tiny-initramfs.bbappend

```diff
@@ -0,0 +1 @@
+include ${@bb.utils.contains('DISTRO_FEATURES', 'podman', 'core-image-tiny-initramfs.inc', '', d)}
```

Comment:
This would make the automatic build of an image with this layer not feasible and would require the manual modification of the local.conf file prior to building.
I would leave it as before and just rename the name of the file from cvm-image-azure to core-imageg-tiny-initramfs.bbappend instead.

The idea is that one would only need to add the custom-podman-layer if they really need it to be in their final image. There is actually no need to add this layer and not use it. It has only the custom recipe to make podman work within initramfs system